src/components/jobList/JobList.tsx

```diff
@@ -1,36 +1,43 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
 /* eslint-disable jsx-a11y/img-redundant-alt */
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector } from 'react-redux';
 import { RootStore } from '../../store';
 import { useState } from 'react';
 import ReactPaginate from 'react-paginate';
 import './jobList.scss';
+import Loading from '../loading/Loading';
 
 const JOB_PER_PAGE = 10;
 
 const JobList: React.FC = () => {
   const jobState = useSelector((state: RootStore) => state.job);
 
-  //Set Page Number
   const [currentPageNumber, setCurrentPageNumber] = useState<number>(0);
 
-  //Number of pages based on the number of items
   const pageOld: number = currentPageNumber * JOB_PER_PAGE;
 
   const jobLength: number = jobState.job?.length || 0;
 
-  //Page Count
   const pageCount = Math.ceil(jobLength / JOB_PER_PAGE);
 
-  //Change Page
   const changePage = (selectedItem: { selected: number }) => {
     setCurrentPageNumber(selectedItem.selected);
   };
 
-  return (
+  useEffect(() => {
+    setCurrentPageNumber(0);
+  }, [jobState.job]);
+
+  return jobState.loading ? (
+    <Loading />
+  ) : jobState.job == null ? (
+    <div className='container w-50 alert alert-danger' role='alert'>
+      ERROR
+    </div>
```

Comment:
This does not make sense. 
- I don't think `job` can be null.
- In javascript or typescript avoid equaling by `==`. 
Please remove it.